app/graphql/mutations/current_user/update.rb

```diff
@@ -11,7 +11,7 @@ def resolve(user:)
         return current_user
       end
 
-      def self.accessible?(context)
+      def self.authorized?(object, context)
```

Comment:
@mikevic  - Because you are checking current_user present? the method name should be authenticated?

Reply:
@manikandan0603 - this is actually a method from the library. 

https://graphql-ruby.org/guides#authorization-guides